site/public/scss/_header.scss

```diff
@@ -88,6 +88,25 @@ header {
     }
 }
 
+.header__center__github {
```

Comment:
I had .header__center__logo as it's own thing since it was the only header__center thing but now that we have this too, we should wrap both of them around a .header__center class that share some attributes like position, left: 50%, height, color, and the @media thing.